### PR TITLE
add false positive CVE-2021-27928 for percona-server-8.3

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: percona-server-8.3.advisories.yaml [hunk 1]
```diff
@@ -21,6 +21,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-06-19T14:19:04Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: This vulnerability afffects Percona Server through 2021-03-03. Our package was build using a release from 2024-04-16.
```
Comment (hectorj2f) [inline percona-server-8.3.advisories.yaml:28]: typo: `package was built`

Comment (cpanato) [reply percona-server-8.3.advisories.yaml]: thanks

File: percona-server-8.3.advisories.yaml [hunk 2]
```diff
@@ -21,6 +21,11 @@ advisories:
             componentType: apk
             componentLocation: /.PKGINFO
             scanner: grype
+      - timestamp: 2024-06-19T14:19:04Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
```
Comment (pdeslaur) [inline percona-server-8.3.advisories.yaml:27]: We have a different type to indicate a version mismatch

Comment (cpanato) [reply percona-server-8.3.advisories.yaml]: updated, let me know if that is the correct one

**Review by pdeslaur — APPROVED** [review]
Great!

**Pull request merged**